editor/src/dev/dev.clj

```diff
@@ -17,29 +17,42 @@
             [clojure.string :as string]
             [dynamo.graph :as g]
             [editor.asset-browser :as asset-browser]
+            [editor.buffers :as buffers]
             [editor.changes-view :as changes-view]
+            [editor.code.data :as code.data]
             [editor.collection :as collection]
             [editor.console :as console]
             [editor.curve-view :as curve-view]
             [editor.defold-project :as project]
             [editor.game-object :as game-object]
+            [editor.gl.vertex2 :as vtx]
             [editor.math :as math]
             [editor.outline-view :as outline-view]
             [editor.prefs :as prefs]
             [editor.properties-view :as properties-view]
             [editor.resource :as resource]
             [editor.resource-node :as resource-node]
+            [editor.scene-cache :as scene-cache]
             [editor.util :as eutil]
             [internal.graph.types :as gt]
             [internal.node :as in]
             [internal.system :as is]
             [internal.util :as util]
+            [lambdaisland.deep-diff2 :as deep-diff]
+            [lambdaisland.deep-diff2.puget.color :as puget.color]
+            [lambdaisland.deep-diff2.puget.printer :as puget.printer]
```

Comment:
We leverage the `pprint` functionality of the `deep-diff2` library (based on `puget`). It gives us more control over formatting than we get from the `clojure.pprint` module.